gio-subclass/src/application.rs

```diff
@@ -287,16 +276,15 @@ where
     let imp = application.get_impl();
 
     //TODO: conversion
-    match imp.local_command_line(&wrap, &from_glib_borrow(arguments)){
-        Ok(status) => {
+    match imp.local_command_line(&wrap, &from_glib_borrow(arguments)) {
+        Some(status) => {
             *exit_status = status;
             1
-        },
-        Err(_) =>  0
+        }
+        None => 0,
```

Comment:
Use `glib_ffi::GTRUE` and `glib_ffi::GFALSE` here

Reply:
Ah, didn't know those existed.

Reply:
That's the default case for conversions of arrays of pointers, nothing special you have to do there